tests/storage/migration.go

```diff
@@ -432,6 +432,9 @@ var _ = Describe(SIG("Volumes update with migration", decorators.RequiresTwoSche
 				destDV, metav1.CreateOptions{})
 			Expect(err).ToNot(HaveOccurred())
 			vm := createVMWithDV(srcDV, volName)
+			vmi, err := virtClient.VirtualMachineInstance(ns).Get(context.Background(), vm.Name,
+				metav1.GetOptions{})
```

Comment:
Why this?
In any case, a `Expect(err).ToNot(HaveOccurred())` is missing

Reply:
It is a leftover, I will remove, thx!